lib/src/main/java/com/futurewei/alcor/common/db/ignite/IgniteConfiguration.java

```diff
@@ -94,7 +94,8 @@ public ICacheFactory igniteClientFactoryInstance(){
     private IgniteClient getThinIgniteClient() {
         ClientConfiguration cfg = new ClientConfiguration();
 
-        cfg.setAddresses(host + ":" + port);
+        cfg.setAddresses(host + ":" + port)
+                .setPartitionAwarenessEnabled(true);
```

Comment:
This type of optimization is pretty good. We could list them on the description of the PR, and if the perf improvement number available, we can also list them there too.